resources/views/platform/components/game/game-list.blade.php

```diff
@@ -20,129 +20,123 @@
 ?>
 
 <div>
-    @if ($areFiltersPristine && count($consoles) === 0)
-        <div class="mb-12">
-            <x-empty-state>{{ $noGamesMessage }}</x-empty-state>
-        </div>
-    @else
```

Comment:
The `@if` block was removed. The rest of the changes in this file are whitespace.